File: src/cli/pm_why_command.zig
pub const PmWhyCommand = struct {
    pub fn exec(ctx: Command.Context, pm: *PackageManager, positionals: []const string) !void {
        try WhyCommand.execFromPm(ctx, pm, positionals);
    }
};

const string = []const u8;

const bun = @import("bun");
const WhyCommand = @import("./why_command.zig").WhyCommand;
const Command = bun.cli.Command;
const PackageManager = bun.install.PackageManager;

File: src/cli/pm_why_command.rs
use crate::command::Context as CommandContext;
use crate::why_command::WhyCommand;
use bun_install::PackageManager;

pub struct PmWhyCommand;

impl PmWhyCommand {
    // TODO(port): narrow error set
    pub fn exec(
        ctx: CommandContext,
        pm: &mut PackageManager,
        positionals: &[&[u8]],
    ) -> Result<(), bun_core::Error> {
        WhyCommand::exec_from_pm(ctx, pm, positionals)?;
        Ok(())
    }
}

// ──────────────────────────────────────────────────────────────────────────
// PORT STATUS
//   source:     src/cli/pm_why_command.zig (12 lines)
//   confidence: high
//   todos:      1
//   notes:      thin delegating wrapper; Command.Context mapped to crate::command::Context
// ──────────────────────────────────────────────────────────────────────────
